.travis.yml

```diff
@@ -7,10 +11,13 @@ matrix:
     - php: 5.6
     - php: 7.1
   fast_finish: true
+  allow_failures:
+    - php: 5.3
+      env: OSTICKET_VERSION=1.10.1
 
 install:
-  - wget https://github.com/osTicket/osTicket/archive/v1.10.1.tar.gz -O ot.tgz
```

Comment:
this will not work

must be 1.9.16

Reply:
the firts faild was corrected with #28 i made a pull for this problem url corrected in #28 but still now we have to investigate why travis still faild. see the travis log of my pull to discuss

Reply:
now travis tests for php 5.6 5.4 and 7 works, but still faild in 5.3, seems something decompressing:

PHP Warning:  include(/home/travis/build/clonemeagain/attachment_preview/tests/plugin.php): failed to open stream: No such file or directory in /home/travis/build/clonemeagain/osticket/include/class.plugin.php on line 280